analyses/molecular-subtyping-LGAT/00-subset-files-for-LGAT.R

```diff
@@ -0,0 +1,44 @@
+# K S Gaonkar 2020
+# Subsets the consensus mutation file to biospecimens where short_histology == LGAT
+
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# get subset folder
+subset_dir<-file.path(root_dir,"analyses","molecular-subtyping-LGAT","lgat-subset")
+
+# clinical file
+clinical<- read_tsv(file.path(root_dir,"data","pbta-histologies.tsv"))
+# consensus mutation data
+consensusMutation <-read_tsv(file.path(root_dir,"data","pbta-snv-consensus-mutation.maf.tsv.gz"))
+
+
+# Filter for LGAT samples
+lgat_wgs_subset <- clinical %>%
+  dplyr::filter(short_histology == "LGAT",
+                sample_type == "Tumor",
+                composition == "Solid Tissue",
+                experimental_strategy == "WGS")
+
+# Filter consensus mutation files for LGAT subset
+consensusMutationSubset <- consensusMutation %>%
+  # find lgat samples
+  dplyr::filter(Tumor_Sample_Barcode %in% lgat_subset$Kids_First_Biospecimen_ID) %>%
+  # look for BRAF V600E mutations and make a column for BRAF_V600E
+  dplyr::filter(Hugo_Symbol == "BRAF" & HGVSp_Short == "p.V600E") %>%
+  # select tumor sample barcode
+  dplyr::select(Tumor_Sample_Barcode,HGVSp_Short) %>% 
+  unique() %>%
+  # join other WGS LGAT samples
+  full_join(lgat_wgs_subset,by=c("Tumor_Sample_Barcode"="Kids_First_Biospecimen_ID")) %>%
```

Comment:
I don't know if you want to join all of the histology data here. I'd probably add a `select()` in here where you only grab the column you need – I'm under the impression that you need the biospecimen IDs in order to "fill out" the rest of the cases _without_ BRAF V600E and not much else at this stage, did I follow correctly?

Reply:
yes that's right, I'll select only the biospecimen IDs here